### PR TITLE
feature/crosswalk

### PR DESCRIPTION
**Pull request by teamagam1**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/teamagam/GimelGimel/pull/67" target="_blank">ReviewNinja

File: app/src/main/assets/GimelGimel/GGBootstrap.js [hunk 1]
```diff
@@ -23,8 +23,12 @@ GG.viewer = new Cesium.Viewer('cesiumContainer', {
     infoBox: false,
     geocoder: false,
     animation: false,
+    creditContainer: "creditContainer"
 });
 
+//cross-view hack.
```
Comment (BarAgam) [inline app/src/main/assets/GimelGimel/GGBootstrap.js]: Make a detailed comment !fix

File: app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/GGMapView.java [hunk 2]
```diff
@@ -37,5 +37,15 @@
      * @param inState a {@link Bundle} object to retrieve the data from it.
      */
     void restoreViewState(Bundle inState);
+
+    /**
+     * pauses the view. needed for pausing javascript timers.
```
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/GGMapView.java]: Delete !fix

File: app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapView.java [hunk 3]
```diff
@@ -62,20 +63,12 @@
...
+        init();
     }
 
-    public CesiumMapView(Context context, AttributeSet attrs, int defStyle) {
-        super(context, attrs, defStyle);
-        init(attrs, defStyle);
-    }
 
     @Override
     public void onDetachedFromWindow() {
```
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapView.java]: move thid method downwards

File: app/src/main/res/layout/fragment_cesium.xml [hunk 4]
```diff
@@ -8,32 +8,33 @@
     <com.teamagam.gimelgimel.app.view.viewer.cesium.CesiumMapView
         android:id="@+id/gg_map_view"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"/>
+        android:layout_height="match_parent"
+        android:hardwareAccelerated="true"/>
```
Comment (BarAgam) [inline app/src/main/res/layout/fragment_cesium.xml:12]: Auto-indent this file !Fix

File: app/src/main/assets/GimelGimel/GGBootstrap.js (hunk 1 above)
Comment (teamagam1) [inline app/src/main/assets/GimelGimel/GGBootstrap.js]: !fixed

File: app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/GGMapView.java (hunk 2 above)
Comment (teamagam1) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/GGMapView.java]: !fixed

**Pull request merged**